cpp/unreal_projects/SpearSim/Source/SpearSim/SpearSim.Build.cs

```diff
@@ -16,6 +16,7 @@ public SpearSim(ReadOnlyTargetRules Target) : base(Target)
         PCHUsage = ModuleRules.PCHUsageMode.Default;
         PrivatePCHHeaderFile = "";
         bUseUnity = false;
+        bEnableUndefinedIdentifierWarnings = false;
```

Comment:
See comment above.

Reply:
I added a comment as to why this param is required.